### PR TITLE
Add Ctrl+F shortcut to open search box

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- For #2447

**Details of Changes**
Add details of changes here.
Code taken from #3558
- Adds "Ctrl + F" keyboard shortcut to open the searchbox and "Esc" to close it

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by Jaiganeshkumaran** [comment]
Does it focus on the last focused element after you press esc? There was this bug in the original PR which I couldn't fix.

**Comment by gave92** [comment]
> Does it focus on the last focused element after you press esc? There was this bug in the original PR which I couldn't fix.

Code is the taken straight from #3558, has the same issue ;)

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -499,7 +499,7 @@
     <value>Refresh (F5)</value>
   </data>
   <data name="NavSearchButton.ToolTipService.ToolTip" xml:space="preserve">
-    <value>Search</value>
+    <value>Search (Ctrl + F)</value>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:502]:
```suggestion
    <value>Search (Ctrl+F)</value>
```

File: Files/UserControls/NavigationToolbar.xaml [hunk 2]
```diff
@@ -912,6 +918,9 @@
                 Style="{StaticResource ToolBarButtonStyle}"
                 ToolTipService.ToolTip="Search">
```
Comment (yaichenbaum) [inline Files/UserControls/NavigationToolbar.xaml:919]:
```suggestion
                ToolTipService.ToolTip="Search (Ctrl+F)">
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**